src/main/java/cloud/fogbow/ras/api/http/request/Compute.java

```diff
@@ -155,6 +159,7 @@ public void pauseCompute(
             throws FogbowException {
 
         try {
+        	// TODO refactor this message
```

Comment:
Please, check the ToDos and implement them.

Reply:
Done